libs/components/forms/src/lib/modules/input-box/input-box.component.html

```diff
@@ -184,23 +181,24 @@
     [errors]="controlDir?.errors"
     [labelText]="labelText"
     [ngClass]="{ 'sky-screen-reader-only': errorsScreenReaderOnly() }"
-    [showErrors]="controlDir?.touched || controlDir?.dirty"
```

Comment:
Most of the changes in this PR are linting issues, I'm going to mark the relevant changes

Reply:
Checking for either "dirty" OR "touched" is an Angular-prescribed condition for when to show error messages. The issue is more than likely with our form components and when we decide to mark a control as "dirty". According to Angular, a form control should be marked as "dirty" when the value has changed. Did you dive into the affected components to see if they're marking the control as "dirty" improperly?

> When the user changes the value in the watched field, the control is marked as "dirty"

See: https://v17.angular.io/guide/form-validation

Reply:
after confirming with design here are how our first class errors are meant to react after this story:

required: touched
maxlength & file attachment errors: dirty
all thers: touched && dirty